src/controllers/userController.js

```diff
@@ -19,5 +19,32 @@ const User = {
     const data = { ...userData, token };
     return sendResult(res, 201, 'Account created successfully', data);
   },
+  async login(req, res) {
+    const { email, password } = req.body;
+    if (!email || !password || !(email && password)) {
+      return sendResult(res, 401, 'Both email and password are required');
+    }
+    const userInfo = await db.Users.findOne({
+      where: {
+        email,
+      },
+    });
+    if (!userInfo) {
+      return sendResult(res, 401, 'Univerifed account or the email and/or password is invalid');
+    }
+
+    const comfirmPass = helpers.comparePassword(password, userInfo.password);
+    if (comfirmPass) {
+      const token = helpers.createToken(userInfo.id, userInfo.email, false);
+      const {
```

Comment:
use basic syntax to destructure your object like `const {password,...data} = user` to remove password or `const {...data, email} = user` to add email.
or even better this way `const {password,...data} = {...data, email}` to remove and update at the same time